deploy.sh

```diff
@@ -120,11 +120,7 @@ function _deploy() {
           jobs=${docker_image_tag} --local --output yaml | kubectl apply -n $namespace -f -
 
   # Apply non-image specific config
-  kubectl apply \
-    -f config/kubernetes/${environment}/service.yaml \
-    -f config/kubernetes/${environment}/ingress.yaml \
-    -f config/kubernetes/${environment}/secrets.yaml \
-    -n $namespace
+  kubectl apply -f config/kubernetes/${environment} -n $namespace
```

Comment:
This will break things - we can't include deployment.yml here because this loads and applies the file from disk, reverting the tag that's applied dynamically on line 120. Please can we put this back how it was?